vnext/Microsoft.ReactNative/Networking.idl

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Let's not call this `Networking`. It's inaccurate and can be confusing due to other networking protocols not involved like WebSockets.

I would suggest using a name that contains `HTTP`.